src/EnergyPlus/ChilledCeilingPanelSimple.hh

```diff
@@ -0,0 +1,347 @@
+	extern Array1D_bool CheckEquipName;
+	extern Array1D_bool SetLoopIndexFlag; // get loop number flag
+
+	//SUBROUTINE SPECIFICATIONS FOR MODULE Simple Chilled Ceiling Panel
+
+	// Types
+
+	struct CoolingPanelParams
+	{
+		// Members
+		std::string EquipID;
+		int EquipType;
+		std::string Schedule;
+		Array1D_string SurfaceName;
+		Array1D_int SurfacePtr;
+		int ZonePtr;
+		int SchedPtr;
+		int WaterInletNode;
+		int WaterOutletNode;
+		int TotSurfToDistrib;
+		int ControlCompTypeNum;
+		int CompErrIndex;
+		int ControlType;
+		std::string ColdSetptSched;
+		int ColdSetptSchedPtr;
+		int CondCtrlType;
+		Real64 CondDewPtDeltaT;
+		int CondErrIndex;
+		Real64 ColdThrottlRange;
+		Real64 RatedWaterTemp;
+		Real64 RatedCapacity;
+		Real64 UA;
+		Real64 Offset;
+		Real64 WaterMassFlowRate;
+		Real64 WaterMassFlowRateMax;
+		Real64 RatedWaterFlowRate;
+		Real64 WaterVolFlowRateMax;
+		Real64 WaterInletTempStd;
+		Real64 WaterInletTemp;
+		Real64 WaterInletEnthalpy;
+		Real64 WaterOutletTempStd;
+		Real64 WaterOutletTemp;
+		Real64 WaterOutletEnthalpy;
+		Real64 RatedZoneAirTemp;
+		Real64 FracRadiant;
+		Real64 FracConvect;
+		Real64 FracDistribPerson;
+		Array1D< Real64 > FracDistribToSurf;
+		Real64 TotPower;
+		Real64 Power;
+		Real64 ConvPower;
+		Real64 RadPower;
+		Real64 TotEnergy;
+		Real64 Energy;
+		Real64 ConvEnergy;
+		Real64 RadEnergy;
+		int LoopNum; // plant loop index
+		int LoopSideNum; // plant loop side index
+		int BranchNum; // plant loop branch index
+		int CompNum; // plant loop component index
+		int CoolingPanelLoadReSimIndex;
+		int CoolingPanelMassFlowReSimIndex;
+		int CoolingPanelInletTempFlowReSimIndex;
+
+		// Default Constructor
+		CoolingPanelParams() :
+			EquipType( 0 ),
+			ZonePtr( 0 ),
+			SchedPtr( 0 ),
+			WaterInletNode( 0 ),
+			WaterOutletNode( 0 ),
+			TotSurfToDistrib( 0 ),
+			ControlCompTypeNum( 0 ),
+			CompErrIndex( 0 ),
+			ControlType( 0 ),
+			ColdSetptSchedPtr( 0 ),
+			CondCtrlType( 0 ),
+		    CondDewPtDeltaT( 0.0 ),
+			CondErrIndex( 0 ),
+			ColdThrottlRange( 0.0 ),
+			RatedWaterTemp( 0.0 ),
+			RatedCapacity( 0.0 ),
+			UA( 0.0 ),
+			Offset( 0.0 ),
+			WaterMassFlowRate( 0.0 ),
+			WaterMassFlowRateMax( 0.0 ),
+			RatedWaterFlowRate( 0.0 ),
+			WaterVolFlowRateMax( 0.0 ),
+			WaterInletTempStd( 0.0 ),
+			WaterInletTemp( 0.0 ),
+			WaterInletEnthalpy( 0.0 ),
+			WaterOutletTempStd( 0.0 ),
+			WaterOutletTemp( 0.0 ),
+			WaterOutletEnthalpy( 0.0 ),
+			RatedZoneAirTemp( 0.0 ),
+			FracRadiant( 0.0 ),
+			FracConvect( 0.0 ),
+			FracDistribPerson( 0.0 ),
+			TotPower( 0.0 ),
+			Power( 0.0 ),
+			ConvPower( 0.0 ),
+			RadPower( 0.0 ),
+			TotEnergy( 0.0 ),
+			Energy( 0.0 ),
+			ConvEnergy( 0.0 ),
+			RadEnergy( 0.0 ),
+			LoopNum( 0 ),
+			LoopSideNum( 0 ),
+			BranchNum( 0 ),
+			CompNum( 0 ),
+			CoolingPanelLoadReSimIndex( 0 ),
+			CoolingPanelMassFlowReSimIndex( 0 ),
+			CoolingPanelInletTempFlowReSimIndex( 0 )
+		{}
+
+		// Member Constructor
```

Comment:
@Myoldmopar So, basically remove everything from this line to the end of this particular definition?  This shows my lack of C++ knowledge because I have no idea what each of these different constructor sections does (still don't) and was simply following examples from modules I was familiar with.  Any ideas on where to find good descriptions of these?